test/conformance/kernel/urKernelSetArgSampler.cpp

```diff
@@ -36,10 +95,13 @@ struct urKernelSetArgSamplerTest : uur::urBaseKernelTest {
 
     ur_sampler_handle_t sampler = nullptr;
 };
+
 UUR_INSTANTIATE_KERNEL_TEST_SUITE_P(urKernelSetArgSamplerTest);
 
-TEST_P(urKernelSetArgSamplerTest, Success) {
-    ASSERT_SUCCESS(urKernelSetArgSampler(kernel, 2, nullptr, sampler));
+TEST_P(urKernelSetArgSamplerTest, SuccessWithProps) {
+    ur_kernel_arg_sampler_properties_t props{
+        UR_STRUCTURE_TYPE_KERNEL_ARG_SAMPLER_PROPERTIES, nullptr};
+    ASSERT_SUCCESS(urKernelSetArgSampler(kernel, 2, &props, sampler));
```

Comment:
What's the meaning of the `2` here and could it be a named variable?

Reply:
Thanks for pointing that, it should be the `arg_index` but I didn't notice that it is a magic number so changed that.